hw5_Zhurylo/js/index.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('DOMContentLoaded', function (){
+    const operation = prompt('write which operation: add, sub, mult, div?');
+    if (operation !== 'add' && operation !== 'sub' && operation !== 'mult' && operation !== 'div') {
```

Comment:
This check is redundant as far as it could be `else block` after all operation check